doc/integrations/postgres.ipynb

```diff
@@ -0,0 +1,227 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f38367ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pytest -x src/tests/integration/test_postgreSQL.py  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "944c80e1",
+   "metadata": {},
+   "source": [
+    "Verify that postgres is running and is ready to accept new connections in the docker terminal. After this if everything goes well it will show 3 passed testcases. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74fb3ae2",
+   "metadata": {},
+   "source": [
+    "### Possible Errors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a48a153",
+   "metadata": {},
+   "source": [
+    "If you face the issue of sql module being not recognized within folder make sure to include it's path in conftest.py."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e241d27",
+   "metadata": {},
+   "source": [
+    "```{Tip}\n",
+    "Docker Error: docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))\n",
+    "```\n",
+    "This error typically occurs when the Docker daemon is not running."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44114c2a",
+   "metadata": {},
+   "source": [
+    "Once the Docker daemon is running, you should be able to proceed with running the PostgreSQL integration tests."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d477fed0",
+   "metadata": {},
+   "source": [
+    "Note: This command will execute the test_postgreSQL.py file, which contains the PostgreSQL-specific integration tests.\n",
+    "<br>Note: We should ensure that by the time the test is running, an instance of each of the tested databases is also running.\n",
+    "<br>\n",
+    "Note: that if you're running pytest with multiple test files or directories, the -x option will stop the test run as soon as the first failure is encountered across any of the specified files or directories.\n",
+    "\n",
+    "```{Tip}\n",
+    "Make sure you have all the imports in place and are of same version as above (Look at setup.py in repo) otherwise let us know if you have issues [message us on Slack.](https://ploomber.io/community)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42349e2d",
+   "metadata": {},
+   "source": [
+    "To skip the tests from test_generic_db_operations.py and only run the PostgreSQL tests, you can run the below script."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40eb5bf2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pytest -x src/tests/integration/test_generic_db_operations.py -k \"ip_with_postgreSQL\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08888580",
+   "metadata": {},
+   "source": [
+    "```{tip} If you have issues [message us on Slack.](https://ploomber.io/community) ```"
```

Comment:
This is not displayed correctly.
<img width="704" alt="Screenshot 2023-06-29 at 11 55 42 AM" src="https://github.com/ploomber/jupysql/assets/29096135/e5c31bd7-aed6-4324-b9a3-ef254fb6c3fc">

Reply:
Fixed.